### ISSUE TITLE
ACC on ImageNet

### ISSUE THREAD

**Issue by zhangdan8962** [k-han/AutoNovel#9]
Hi,

You mentioned in the paper that ACC result for Imagenet is from same 30-class split that was used for training.

I am wondering how ACC would be if I use another split instead of the one used for training.

**Comment by zhangdan8962** [comment]
I got another question and I am wondering if you have any insights or thoughts.

Since your model is guided by ranking statistics, I was assuming if I took the output feature Z(512-d)from Resnet using your pretrained model on Imagenet subsetA, I can directly tell if two images are from same class by indices of top5 dimensions. So I did following experiment:

1. Generate ground truth matrix G(37885,37885) in which G(i,j)=1  if both images are from same class.
2. Generate prediction matrix P(37885,37885) in which P(i,j) = 1  if indices of top5 dimensions for both images are same set(same method used in your paper).

However, It is surprising that these two matrix are quite different given the fact that your model achieves 82% ACC on softmax layer.

Thank you in advance for any help!

**Comment by jingzhengli** [comment]
Hi, I think the meaning of "ranking statisticscal" is not " indices of top5 dimensions"
